src/network/networkctl.c

```diff
@@ -2479,21 +2479,24 @@ static int link_status(int argc, char *argv[], void *userdata) {
         return 0;
 }
 
-static char *lldp_capabilities_to_string(uint16_t x) {
+static char *lldp_capabilities_to_string(uint16_t x, bool dots) {
         static const char characters[] = {
                 'o', 'p', 'b', 'w', 'r', 't', 'd', 'a', 'c', 's', 'm',
         };
         char *ret;
-        unsigned i;
+        unsigned i, j;
```

Comment:
maybe switch to size_t while you are at it, after all ELEMENTSOF() actually returns that